docs-ref-mapping/reference-unified.yml

```diff
@@ -953,16 +958,16 @@
         - azure-arm-mobileengagement        
     - name: Monitor
       uid: azure.nodejs.sdk.landingpage.services.monitor
-      href: ~/docs-ref-services/{moniker}/monitor.md
+      landingPageType: Service
```

Comment:
```suggestion
      href: ~/docs-ref-services/{moniker}/monitor.md
      landingPageType: Service
```

Reply:
```suggestion
```

Reply:
You have a href, href and landingPageType: Service are mutually exclusive.